src/main/java/seedu/address/ui/ViewWindow.java

```diff
@@ -47,16 +47,16 @@ public ViewWindow(Person person) {
         super(FXML);
         this.person = person;
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        email.setText(person.getEmail().value);
-        telegram.setText(person.getTelegram().value);
+        phone.setText("Phone Number: " + person.getPhone().value);
```

Comment:
Will it be possible to change the colour of the labels (i.e. change the colour of `Phone Number: ` and so on) so that it is slightly easier to read? I feel now since everything is the same size, it is a bit hard to distinguish between the label and the value in one look. However, this is just my opinion!

Reply:
Ok, I have an idea on how to do it and will try it out